coalib/parsing/FilterHelper.py

```diff
@@ -1,10 +1,11 @@
 from coalib.parsing.InvalidFilterException import InvalidFilterException
 from coalib.parsing.filters import available_filters
 from coalib.parsing.DefaultArgParser import default_arg_parser
+from collections import OrderedDict
```

Comment:
Please move this import to the top and give a space to confirm to our standards. Standard library imports should be topmost. Refer https://github.com/coala/coala/blob/d885c4ad2729282cf43083e08be041cf70e458bf/coalib/io/FileFactory.py